contract_test/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
+jsonschema
+pytest
```

Comment:
We are not using pytest for the main repo. I want to avoid using two different python test frameworks in the same repo. If pytest is better - we should migrate existing tests from unittest to pytest. Otherwise stick to unittest. Can you do a comparison between the two and propose which one we should use?

Reply:
fixed. Now only `unittest`